forecasts.js

```diff
@@ -235,21 +234,12 @@ forecasts.forecastForRegionId = function(regionId, onForecast) {
     if (!regionDetails) {
         winston.warn('invalid regionId: ' + regionId);
         process.nextTick(function() { onForecast(null); } );
-    } else if (regionDetails.provider === 'uac' && forecasts.forecastGenerationCount % 6 !== 0) {
-        // HACK for uac issue where they are blocking our fetches accidentally if they happen too often; so only actually
-        // fetch it every N times
-        
-        if (forecasts.mostRecentForecasts[regionId]) {
-            forecast = forecasts.mostRecentForecasts[regionId];
-            winston.info('using cached value for region: ' + regionId + '; forecast: ' + JSON.stringify(forecast));
-        }
-        
-        process.nextTick(function() { onForecast(forecast); } );
-    } else {
+    }
+    else {
```

Comment:
code style nit: let's have the two lines above on one line, i.e. 
`} else {`
to match the style of the rest of the code.

Reply:
Sounds great. Sorry about that.